cmd/go-quai/start.go

```diff
@@ -85,15 +85,21 @@ func runStart(cmd *cobra.Command, args []string) error {
 	}
 
 	logLevel := cmd.Flag(utils.LogLevelFlag.Name).Value.String()
-	// create instance of consensus backend
-	var nodeWG sync.WaitGroup
-	consensus, err := utils.StartQuaiBackend(ctx, node, logLevel, &nodeWG)
+
+	var startingExpansionNumber uint64
+	if viper.IsSet(utils.StartingExpansionNumberFlag.Name) {
+		startingExpansionNumber = viper.GetUint64(utils.StartingExpansionNumberFlag.Name)
```

Comment:
Maybe do a sanity check on the number here (or later)

Reply:
added  a sanity check for expansion number